backend/Controllers/LocationController.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Services;
+
+namespace WhaleSpotting.Controllers {
+  [ApiController]
+  [Route("/locations")]
+  public class LocationController : ControllerBase {
+    private readonly ILocationService _locations;
+
+    public LocationController(
+      ILocationService locations
```

Comment:
In C#, prefer brackets on the next lines:
```suggestion
    public LocationController
    (
      ILocationService locations
```